src/app/text-display/text-display.component.ts

```diff
@@ -0,0 +1,46 @@
+import {Component, Input} from '@angular/core';
+
+export type TextDisplayEditCallbackFn = (saved: boolean) => void;
+
+@Component({
```

Comment:
+doc comment with a short explanation of what this component does

Reply:
Alright, documentation added.